### PR TITLE
fix(releases): change project link to btn

### PR DESCRIPTION
**Pull request by doralchan**
**Revised CTA for breakpoints:**
Screen Shot 2021-01-20 at 9 25 01 AM
Screen Shot 2021-01-20 at 9 25 15 AM
Screen Shot 2021-01-20 at 9 25 28 AM
Screen Shot 2021-01-20 at 9 25 39 AM

### CODE REVIEW

File: src/sentry/static/sentry/app/views/releases/list/releaseHealth/content.tsx [hunk 1]
```diff
@@ -252,41 +257,39 @@ const CollapseButtonWrapper = styled('div')`
 `;
 
 const ProjectRow = styled(PanelItem)`
-  padding: 10px ${space(2)};
-  max-height: 41px;
+  padding: 8px ${space(2)};
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/views/releases/list/releaseHealth/content.tsx:260]:
```suggestion
  padding: ${space(1)} ${space(2)};
```

**Comment by priscilawebdev** [comment]
it does not look good on smaller devices. Please see:

**Before:**
image

**After:** 
image

**Comment by priscilawebdev** [comment]
image

File: src/sentry/static/sentry/app/views/releases/list/releaseHealth/content.tsx [hunk 2]
```diff
@@ -72,8 +73,8 @@ const Content = ({
               </DailyColumn>
             </React.Fragment>
           )}
-          <CrashesColumn>{t('Crashes')}</CrashesColumn>
-          <IssuesColumn>{t('New Issues')}</IssuesColumn>
+          <RightColumn>{t('Crashes')}</RightColumn>
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/views/releases/list/releaseHealth/content.tsx:76]: I don't think we should call it "RightColumn", as the columns are not visually separated

File: src/sentry/static/sentry/app/views/releases/list/releaseHealth/content.tsx [hunk 3]
```diff
@@ -188,25 +185,33 @@ const Content = ({
                     )}
                   </DailyColumn>
 
-                  <CrashesColumn>
+                  <RightColumn>
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/views/releases/list/releaseHealth/content.tsx:188]: shouldn't we use a SessionColumn here as we display session data?

File: src/sentry/static/sentry/app/views/releases/list/releaseHealth/content.tsx [hunk 4]
```diff
@@ -252,41 +257,39 @@ const CollapseButtonWrapper = styled('div')`
...
+  @media (min-width: ${p => p.theme.breakpoints[3]}) {
+    grid-template-columns: 1fr 0.8fr 1fr 1fr 0.5fr 0.5fr 0.5fr;
   }
 `;
 
 const Column = styled('div')`
   ${overflowEllipsis};
-  height: 20px;
   line-height: 20px;
+  display: flex;
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/views/releases/list/releaseHealth/content.tsx:289]: text-overflow: ellipsis won't work on flex container

**Comment by matejminar** [comment]
Would it make sense to make version clickable?

If we have a single project in the global header we could point directly to the right release detail.

If there are multiple projects selected it could point to the release detail project picker:
image

That said, I'd also leave those new "View" buttons there.

**Comment by doralchan** [comment]
@matejminar I'd rather not change the flow by changing the global header in this branch. This is more of a UX question I'd rather resolve after this.

**Comment by priscilawebdev** [comment]
do we want to display only the project's name and the crash-free session on smaller devices?

**Before:**

image



**After:**

image


I like more the previous version as it was more complete.

**Comment by priscilawebdev** [comment]
can you please check this issue?

image

**Comment by priscilawebdev** [comment]
Could you please check this issue?

image

**Comment by priscilawebdev** [comment]
the card in this breakpoint looks so empty. Please are you sure about this?

image

**Review by robinrendle — APPROVED** [review]
👍 lgtm

**Pull request merged**